WofiEmoji

U-L-M-S opened the pull request:
a script that let you pick a emoji and save it in your clipboard.

U-L-M-S committed:
add Wofi-Emoji
It's a script that let you pick any emoji and saves it on your clipboard. You can add more emojis and even change the name of the emojis in the script. shortcut: SUPER+E

U-L-M-S committed:
Merge branch 'JaKooLit:main' into main

U-L-M-S committed:
add WofiEmoji
It's a script that let you pick any emoji and saves it on your clipboard. You can add more emojis and even change the name of the emojis in the script. shortcut: SUPER+ALT+E. Checkout the WofiEmoji.png for an preview

JaKooLit commented:
Ive tried and not all emoji are showing. Perhaps additional fonts needed. Kindly add on install script fonts needed.

JaKooLit committed:
Update WofiEmoji.sh

Added height and width option. If to use wofi, it changed depending on waybar layout style change hence to use the WofiBig Configs

JaKooLit commented:
Im merging, however, needed additional fonts required
